src/django_pg_returning/manager.py

```diff
@@ -39,7 +39,7 @@ def _insert(self, objs, fields, return_id=False, raw=False, using=None, ignore_c
         query.insert_values(fields, objs, raw=raw)
 
         self.model._insert_returning_cache = self._execute_sql(query, return_fields, using=using)
-        return self.model._insert_returning_cache.values_list('id', flat=True) if return_id else None
+        return self.model._insert_returning_cache.values_list(self.model._meta.pk.column, flat=True) if return_id else None
```

Comment:
I suppose return_id should be renamed to return_pk

Reply:
Hello - return_id is part of the django query api and as such it would be best to also use the same parameter name.